src/main/java/com/e2e/exercises/exercise1/BaseTest2.java

```diff
@@ -0,0 +1,101 @@
+package com.e2e.exercises.exercise1;
```

Comment:
Duplicate file BaseTest2?

Reply:
Have written 2 classes, one with JSON object from net.minidev.json and another one with org.json